pwndbg/commands/elf.py

```diff
@@ -32,8 +43,17 @@ def elfsections() -> None:
 
         sections.sort()
 
+        # Header
+        print("ADDRESS", end="\t\t")
+        if pie_relocation:
+            print("RELOCATED", end="\t\t\t")
```

Comment:
Can you show how this looks like?

Shall we maybe use the `tabulate` library we use in different areas of Pwndbg instead of tabulating things manually? Can you check if this would be better?

This would be something like:
```py
from tabulate import tabulate
print(tabulate(table, headers))
```

See how it is used in pwndbg/command/tls.py